auth_matrix/admin.py

```diff
@@ -61,38 +66,24 @@ class Meta:
         fields = ("name",)
         export_order = fields
 
-    def dehydrate(self, group):
-        data = super().dehydrate(group)
+    def after_export(self, queryset, data, *args, **kwargs):
         permission_names = [permission.name for permission in Permission.objects.all()]
```

Comment:
```suggestion
        permission_names = Permission.objects.values_list("name", flat=True)
```
you can do this in a single optimized query :)